.prettier.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  overrides: [
+    {
+      files: "**/*.sol",
+      options: {
+        printWidth: 120,
+        tabWidth: 2,
+      },
+    },
+    {
+      files: ["**/*.js", "**/*.ts", "**/*.vue"],
```

Comment:
Should have jsx/tsx instead of vue files. But maybe we just remove this key so it runs against all supported files? e.g. JSON and markdown too